setup.py

```diff
@@ -63,9 +63,6 @@
                         'unicorn>=1.0.2rc1', # see unicorn-engine/unicorn#1100, unicorn-engine/unicorn#1170
 ]
 
-if sys.platform == 'win32':
-    install_requires.append('windows-curses>=2.1.0')
```

Comment:
It was necessary in the previous commit, since pwnlib.term.termcap was still imported and used throughout the codebase. `windows-curses` has the `tigetstr` family of API nulled
https://github.com/zephyrproject-rtos/windows-curses/blob/8256c8970c53c26f9885c799febfeea671394403/term.h#L3-L6